src/docs/analytics.mdx

```diff
@@ -156,15 +157,42 @@ For class based components, the following functions can be used:
 
 You will need to import both `withRouteAnalytics` and `WithRouteAnalyticsProps`. `WithRouteAnalyticsProps` needs to be added to the Props of the class. The functions can then accessed through the Props. `withRouteAnalytics` is called while exporting the class.
 
-Example: 
+Example:
+
 ```tsx
 class Monitors extends AsyncView<Props, State> {
   componentDidMount() {
     this.props.setEventNames('monitors.page_viewed', 'Monitors: Page Viewed');
   }
 
- ```
-  
+```
+
+## Button Click Analytics
+
+All in-app button clicks will generate analytics events by default for Reload. As of now, they are not automatically being sent to Amplitude, but you can set it up so that the events are sent there.
+
+Out of the box, each click on a button will track the `text` (aria-label), `priority`, `href`, and `parameterized_path` of the button. You might encounter an issue where some of the data will be an empty string which is a known bug. The event name will be based on the parameterized path , similar to route based analytics (see above for more information) but will be prefixed with `button_click`.
+
+### Customization
+
+There are a few ways to customize the button click analytics:
+
+- `analyticsEventName` - Allows the event to be sent to Amplitude with the given event name. If this is not set, then the event will not be sent to Amplitude
+- `analyticsEventKey` - Overrides the default parameterized path event key and replaces it with the given event key. Unless `analyticsEventName` is also set, this will continue to only send events to Reload, but using the new event key. The parameterized path will still be sent as part of the event, but will not be the event key.
+- `analyticsParams` - Adds additional parameters that will be tracked with each button click
+
+Example:
+
+```tsx
+<RestartButton
+  analyticsEventName="Growth: Guided Tour Restart"
+  analyticsEventKey="growth.guided_tour_restart"
+  analyticsParams={{tour}}
```

Comment:
@roggenkemper I think it might it more clear if you did something like `{{tour: 'issues'}}` that actually shows that it's a string